account_move_ref_document/models/account_move.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import api, models, fields
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    document_id = fields.Reference(
+        selection='_selection_model',
+        string='Document',
+        help="Reference to the document that create this account.move",
```

Comment:
`Reference to the document that create this Account Move`

Reply:
Done